ospfd/ospf_ext.c

```diff
@@ -1705,11 +1705,15 @@ static void ospf_ext_lsa_schedule(struct ext_itf *exti, enum lsa_opcode op)
  * ------------------------------------
  */
 
+/* Check NULL for vty. If vty is not available, dump info via zlog */
 #define check_tlv_size(size, msg)                                              \
```

Comment:
this doesn't seem very useful - it does a test, but then if the test fails, the code just continues with the failed object?
and "Abort" - that's a tough word to use, because we want to skip/ignore an invalid blob of data, not crash/abort if we see one?